src/web/mjs/connectors/To-Corona-Ex.mjs

```diff
@@ -51,30 +51,89 @@ export default class ToCoronaEx extends Connector {
 
     async _getChapters(manga) {
         let chapterList = [];
+        const getToken = await this.getToken();
```

Comment:
The variable has the same name as the member function `getToken`, i recommend to use a more appropriate name